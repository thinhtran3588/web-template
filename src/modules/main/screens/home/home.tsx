import {useRouter} from 'next/router';
import type {NextPage} from 'next';
import {getI18nText} from '@core/helpers/get-i18n-text';
import SITE_I18N_TEXT from '@locales/site.json';

export const Home: NextPage = (): JSX.Element => {
  const router = useRouter();

  return (
    <div className='flex flex-col w-full'>
      <div>{getI18nText(SITE_I18N_TEXT, 'TAB_HOME', router)}</div>
    </div>
  );
};
