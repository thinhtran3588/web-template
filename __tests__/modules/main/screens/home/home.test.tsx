/**
 * @jest-environment jsdom
 */

import React from 'react';
import {render} from '@test/test-utils';
import {Home} from '@main/screens/home/home';

describe('Home', () => {
  it('renders successfully', async () => {
    const {container} = render(<Home />);

    expect(container).toMatchSnapshot();
  });
});
