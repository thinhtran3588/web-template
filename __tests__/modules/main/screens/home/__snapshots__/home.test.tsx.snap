// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home renders successfully 1`] = `
<div>
  <div
    class="flex flex-col w-full"
  >
    <div>
      Home
    </div>
  </div>
</div>
`;
